### DISCUSSION TITLE
Band Math Expressions

### DISCUSSION THREAD

**Discussion by abillits** [developmentseed/titiler#452] [Q&A] [answered]
Perhaps this belongs over on the rio-tiler github but I'm asking here in case there are limitations imposed by Titiler.

Is there a list of what's possible with band math expressions? I gather that basic addition, subtraction, division, and multiplication are possible. However, are there any more advanced functionalities?

Specifically I'm interested in something like: max(b4,b5,b6,b13) that would return that max value out of those bands. Similar for minimum.

**Comment by vincentsarago** [comment, marked as answer]
@idtstudios, yes this should be in rio-tiler but it's fine I should probably make the doc clearer for this. 

band math expression using `numexpr` so here are the operator/function available https://numexpr.readthedocs.io/en/latest/user_guide.html#supported-operators

Sadly `max` is not part of the available function (see https://github.com/pydata/numexpr/issues/86)

**Reply by abillits** [reply]
Thanks for the response. This is what I needed. It looks like I might be able to do max/min via the where function.
